File: manual-stress.c
#include <stdlib.h>
#include <stdio.h>
#include <string.h>

void stress_cpu() {
    while (1) {
        for (int i = 0; i < 10000000; ++i) {
            int result = 1 + 1;
            result++; 
        }
    }
}

void stress_ram() {
    unsigned long chunk_size = 100 * 1024 * 1024; 
    unsigned long allocation_count = 0;
    
    while (1) {
        void* memory = malloc(chunk_size);
        if (memory == NULL) {
            break;
        }
        
        memset(memory, 0, chunk_size);
        allocation_count++;
    }
    
    printf("Allocated %lu chunks of %luMB\n", allocation_count, chunk_size / (1024 * 1024));
}

int main() {
    printf("Starting CPU and RAM stress test. Press Ctrl+C to stop.\n");
    
    while (1) {
        stress_cpu();
        stress_ram();
    }
    
    return 0;
}

File: manual_stress.py
#!/usr/bin/python3
# Code By Smukx : https://github.com/Whitecat18/Linux-Attacks
# stress the CPU and RAM until it is manually stopped using Ctrl+C:
# Install this advance module called psutil ` pip install psutil`

import psutil
import time

def stress_cpu(duration):
    start_time = time.time()
    end_time = start_time + duration
    
    while time.time() < end_time:

        for _ in range(10**7):
            _ = 1 + 1

def stress_ram(duration):
    start_time = time.time()
    end_time = start_time + duration
    

    data = []
    while time.time() < end_time:
        data.append(' ' * 1024) 

    data.clear()

try:
    duration = int(input("Enter stress duration in seconds: "))
    print("Starting CPU and RAM stress test. Press Ctrl+C to stop.")
    
    stress_cpu(duration)
    stress_ram(duration)

except KeyboardInterrupt:
    print("Stress test stopped manually.")
